requirements.txt

```diff
@@ -4,7 +4,7 @@ flask==0.10.1
 -e git+ssh://git@github.com/closeio/mongoengine.git@b62ad0226336d6db39c85ce2cff49aeb94cc919e#egg=mongoengine-dev
 -e git+ssh://git@github.com/closeio/flask-mongoengine.git@0c2cc30ec98154bb2eb4499efada65239050025d#egg=flask-mongoengine
 Flask-SQLAlchemy==2.1
-phonenumbers==7.2.6
+phonenumbers==8.9.2
```

Comment:
Should this match the version we use in Close.io so that we don't introduce changes that work here, but don't work in our app?

Reply:
Can we just use the latest version in Close.io also? The main reason I upgraded it here is so that tests work with all Python versions.